### ISSUE TITLE
No networking info and graphs on the networking tab (Ubuntu Server 17.04)

### ISSUE THREAD

**Issue by ghost** [cockpit-project/cockpit#7889]
![image](https://user-images.githubusercontent.com/6624832/31616425-faacf8e6-b28c-11e7-8f89-7a41c0ab1d88.png)
When I click on Add Bond it shows Oops and it displays the following error on the chrome dev tools window.

```
network.js:431 Uncaught TypeError: Cannot read property 'split' of undefined
    at n (network.js:431)
    at F (network.js:435)
    at l.$.at_least_version (network.js:709)
    at j.update (network.js:1829)
    at j.enter (network.js:1785)
    at HTMLDivElement.<anonymous> (network.js:998)
    at HTMLDivElement.dispatch (jquery.js:2460)
    at HTMLDivElement.m.handle (jquery.js:2274)
    at Object.trigger (jquery.js:2414)
    at HTMLDivElement.<anonymous> (jquery.js:2785)
```

**Additional info (Worth mentioning)**
I double checked if network-manager was installed and running and found out that systemd-networkd wasn't running on my system I am not sure if this had anything to do with this bug.
After that I ran networkctl and it showed all my network adapters correctly

![image](https://user-images.githubusercontent.com/6624832/31616620-789093bc-b28d-11e7-89c9-28fb457adb7c.png)

**Comment by CodeSurfing145** [comment]
Add VLAN also throws an error
```
network.js:40 Uncaught TypeError: Cannot read property 'choice' of undefined
    at r (network.js:40)
    at q.update (network.js:2132)
    at q.enter (network.js:2108)
    at HTMLDivElement.<anonymous> (network.js:998)
    at HTMLDivElement.dispatch (jquery.js:2460)
    at HTMLDivElement.m.handle (jquery.js:2274)
    at Object.trigger (jquery.js:2414)
    at HTMLDivElement.<anonymous> (jquery.js:2785)
    at Function.each (jquery.js:189)
    at p.fn.init.each (jquery.js:45)
```

**Comment by petervo** [comment]
What version of cockpit are you running?

**Comment by CodeSurfing145** [comment]
Version 151.

**Comment by CodeSurfing145** [comment]
After all this wasn't a issue caused by cockpit, it had to do with the fact that my Ubuntu 17.04 didn't use network manager for managing the interfaces, I had to install network-manager then set the config to `managed=true` and also run
```
sudo touch /etc/NetworkManager/conf.d/10-globally-managed-devices.conf 
sudo service network-manager restart
```

Then I relogged into cockpit and all worked fine!!

**Comment by petervo** [comment]
@CodeSurfing145 thanks for tracking down the issue. I'm reopening to see if there is anything we can do in cockpit to handle this more gracefully.

@mvollmer, is there anything we can do to detect this, rather than having the JS throw an error?

**Comment by martinpitt** [comment]
I believe I sufficiently addressed this in PR #9624.
